src/Symfony/Component/Mailer/Bridge/Mailjet/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Add `RemoteEvent` and `Webhook` support
+ * Fix `TrackClicks` and `TrackOpens` parameters
```

Comment:
This line must be reverted, changelogs for patch releases are drafted automatically. The components' changelog files do only contain features/deprecations added in minor releases.

Reply:
I've reverted the line in the changelog.